### PR TITLE
Add a recipe for xcb-proto

### PR DESCRIPTION
**Pull request by ccordoba12**
This is another build dependency of libxcb, which is a runtime dep for Qt5 on Linux.

### CODE REVIEW

**Comment by ccordoba12** [comment]
@ocefpaf, please review also this one. It should be ready too.

File: recipes/xcb-proto/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    # Taken from http://www.linuxfromscratch.org/patches/blfs/svn/xcb-proto-1.12-schema-1.patch
+    - xcb-proto-1.12-schema-1.patch
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - python 2.7*
```
Comment (ocefpaf) [inline recipes/xcb-proto/meta.yaml]: Remove the version here and add py3k on the skip above. `skip: true  # [win or osx or py3k]` should work.

**Comment by ocefpaf** [comment]
LGTM just one minor comment.

The feedstock generation is broken:

```shell
Traceback (most recent call last):
  File "/Users/travis/miniconda/bin/conda-build-all", line 9, in <module>
    load_entry_point('conda-build-all==0.13.3', 'console_scripts', 'conda-build-all')()
  File "/Users/travis/miniconda/lib/python3.5/site-packages/conda_build_all/cli.py", line 85, in main
    b.main()
  File "/Users/travis/miniconda/lib/python3.5/site-packages/conda_build_all/builder.py", line 228, in main
    recipe_metas = self.fetch_all_metas()
  File "/Users/travis/miniconda/lib/python3.5/site-packages/conda_build_all/builder.py", line 152, in fetch_all_metas
    recipe_metas = list_metas(conda_recipes_directory)
  File "/Users/travis/miniconda/lib/python3.5/site-packages/conda_build_all/builder.py", line 78, in list_metas
    packages.append(MetaData(new_root))
  File "/Users/travis/miniconda/lib/python3.5/site-packages/conda_build/metadata.py", line 373, in __init__
    self.parse_again(permit_undefined_jinja=True)
  File "/Users/travis/miniconda/lib/python3.5/site-packages/conda_build/metadata.py", line 387, in parse_again
    self.meta = parse(self._get_contents(permit_undefined_jinja), path=self.meta_path)
  File "/Users/travis/miniconda/lib/python3.5/site-packages/conda_build/metadata.py", line 185, in parse
    (field, res[field].__class__.__name__, path))
RuntimeError: The test field should be a dict, not str in file /Users/travis/build/conda-forge/staged-recipes/recipes/qscintilla2/meta.yaml.
```

I will take a look soon but I suspect is a `conda-build` `conda-build-all` incompatibility again. (Ping @msarahan.)

File: recipes/xcb-proto/meta.yaml (hunk 1 above)
Comment (ccordoba12) [inline recipes/xcb-proto/meta.yaml]: Ok, will do

**Comment by jakirkham** [comment]
Could we please rebase this so it is a bit easier to read the diff?

**Comment by ccordoba12** [comment]
Thanks @ocefpaf!!

**Pull request merged**